ar_exercise/exercise.rb

```diff
@@ -32,27 +32,65 @@ class Apartment < ActiveRecord::Base
 # Use Active record to do the following, and store the results **in a variable**
 # example: get every tenant in the DB
 all_tenants = Tenant.all
+all_apartments = Apartment.all
 
 # get the first tenant in the DB
+first_tenant = Tenant.first
+
 # get all tenants older than 65
+older_tenants = Tenant.where("age > '65'")
+
 # get all apartments whose price is greater than $2300
+expensive_apts = Apartment.where("monthly_rent > '2300'")
+
 # get the apartment with the address "6005 Damien Corners"
+damien_corners = Apartment.where("address = '6005 Damien Corners'")
+
 # get all tenants in that apartment
+damien_corners = Apartment.where("address = '6005 Damien Corners'")[0].tenants
 
```

Comment:
in the line above you already did a query for the object you wanted. Assuming you use `find_by` you could just do

```ruby
damien_corners.tenants
```